src/toolkit/bamtools_convert.cpp

```diff
@@ -509,9 +656,18 @@ void ConvertTool::ConvertToolPrivate::PrintJson(const BamAlignment& a)
                     m_out << '"';
                     ++index;
                     break;
+
+                case (Constants::BAM_TAG_TYPE_ARRAY):
+                    m_out << "[";
```

Comment:
```suggestion
                    m_out << '[';
```
Avoid string literals